src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/vet/FeaturedVideoContainer.java

```diff
@@ -0,0 +1,112 @@
+package com.wikia.webdriver.pageobjectsfactory.componentobject.vet;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import lombok.Getter;
+import org.apache.commons.lang.StringUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+
+import java.io.File;
+
+public class FeaturedVideoContainer extends WikiBasePageObject {
+
+
+  private By videoTitleTextBoxBy = By.cssSelector(".video-title");
+  private By imageTitleTextBoxBy = By.cssSelector(".alt-thumb-name");
+  private By addImageButtonBy = By.cssSelector(".media-uploader-btn");
+  private By addVideoButtonBy = By.cssSelector(".add-video-button");
+  private By addImageModalBy = By.cssSelector("#WMU_divWrapper");
+  private By addVideoModalBy = By.cssSelector("#VideoEmbedBackWrapper");
+  private By displayedTitleTextBoxBy = By.cssSelector(".display-title");
+  private By descriptionTextBoxBy = By.cssSelector(".description");
+
+
+  @Getter
+  public WebElement parentElement;
+
+  private WebElement videoTitleTextBox;
+  private WebElement imageTitleTextBox;
+  private WebElement addVideoButton;
+  private WebElement addImageButton;
+  private WebElement displayedTitleTextBox;
+  private WebElement descriptionTextBox;
+
+
+  public FeaturedVideoContainer(WebDriver driver, WebElement parentElement) {
+    super();
+    this.parentElement = parentElement;
+//    PageFactory.initElements(driver, this);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)